src/main/java/com/project/escape/domain/reservation/user/UserReservationId.java

```diff
@@ -0,0 +1,15 @@
+package com.project.escape.domain.reservation.user;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserReservationId implements Serializable {
+    private Long user;
+    private Long reservation;
```

Comment:
`UserReservation` Entity에 있는 
`user`와 `reservation`의 변수명이 같아야 매핑이 가능했습니다. 
그래서
이곳의 변수명도 `user`, `reservation` 하였습니다.

근데 사실 얘네들이 가르키고 있는건 Id 인데요.
```suggestion
    private Long userId;
    private Long reservationId;
```
이렇게 선언하면 UserReservation에 create힐 때에 SQL에러가 났습니다.

`SQLState: 07001` 이 에러였고
SQL 쿼리에 인자의 수가 맞지 않는 경우에 나는 에러였습니다.
보니까 중복키가 선언되어 에러가 났습니다.

이런 경우에 변수를 어떻게 선택해야할까요?

Reply:
제가 이해한게 맞는지 모르겠지만, UserReservation 테이블을 보면 복합키로 구성되어 있던데요~
여기서 꼭 복합키로 구성되어야 할까? 라는 고민이 들긴 합니다. reservationId가 유니크 하다면 user_id는 pk가 아니라 fk가 되어도 좋을것 같아서요~ 요거 한번 고민해봐주시고 복합키가 아닌 reservationId를 uk + pk 로 가져가면 좀 쉽게 해결이 되지 않을까 생각이 되네요~

Reply:
아 그렇네요. 원래 reservation을 `ManyToOne`으로 설정했다가 OneToOne으로 변경하였는데요. 그냥 fk를 걸면 간단해지네요.

Reply:
해결한 커밋 : https://github.com/f-lab-edu/escape/pull/2/commits/638842264ca6574a1e34dcbe07f243a2a0167583